doozer/doozerlib/release_inspector.py

```
@@ -45,20 +49,47 @@ async def fetch_release_pullspec_from_stream_api(release_name: str, major_minor:
     raise IOError(f"Nightly {release_name} not found in release stream {release_stream_name}")
 
 
-async def introspect_release(pullspec: str, registry_config: str = None) -> Model:
+async def introspect_release(
+    pullspec: str, registry_config: str = None, retries: int = 5, retry_delay: float = 30.0
```

Ruff (0.15.12)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)